content/post/gld.md

```diff
@@ -117,6 +117,8 @@ After careful tuning of false positives, false negatives, and performance, we de
 similarity threshold for our algorithm to 75%, and the hash length to 154 samples.
 Since we discard the text structure by treating sequences as sets, we further calculate the Levenshtein
 distance to the database records matched by Weighted MinHash in order to determine the precise confidence value.
+Note: this does not take into account the linguistic tricks, e.g. negations. Fortunately,
```

Comment:
Add an extra line before the note for better formatting

Also use _Note:_.

I would say it differently, something like: this approach is easily wronged by slight textual modifications on well known licenses which big impacts on their semantics, e.g. adding a "not" in an otherwise well known sentence.

> You may reproduce and distribute copies of the Work or Derivative Works
vs
> You may *not* reproduce and distribute copies of the Work or Derivative Works